bigtable/google/cloud/bigtable/instance.py

```diff
@@ -239,3 +244,219 @@ def list_tables(self):
             result.append(self.table(table_id))
 
         return result
+
+    def create_app_profile(self, app_profile_id, etag='', description='',
+                           ignore_warnings=None, metadata=None,
+                           routing_policy_type=ROUTING_POLICY_TYPE_ANY,
+                           cluster_id=None,
+                           allow_transactional_writes=False):
+        """Creates an app profile within an instance.
+
+        :type: app_profile_id: str
+        :param app_profile_id: The unique name for the new app profile.
+
+        :type: etag: str
+        :param: etag: (Optional) Strongly validated etag for optimistic
+                        concurrency control. Preserve the value returned
+                        from ``GetAppProfile`` when calling
+                        ``UpdateAppProfile`` to fail the request if there
+                        has been a modification in the mean time. The
+                        ``update_mask`` of the request need not include
+                        ``etag`` for this protection to apply.
+
+        :type: description: str
+        :param: description: (Optional) Long form description of the use
+                                case for this AppProfile.
+
+        :type: ignore_warnings: bool
+        :param: ignore_warnings: (Optional) If true, ignore safety checks when
+                                    creating the app profile.
+
+        :type: metadata: [Sequence[Tuple[str, str]]]
+        :param: metadata: (Optional) Additional metadata that is provided to
```

Comment:
I'm not sure what metadata is.  There isn't any metadata in the [protobuf definition](https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/bigtable_instance_admin.proto#L439).

If you don't know what this is, please remove it.